media_driver/agnostic/gen11/codec/hal/codechal_kernel_header_g11.h

```diff
@@ -36,19 +36,27 @@ struct HmeDsScoreboardKernelHeaderG11 {
         struct
         {
             CODECHAL_KERNEL_HEADER hmeDownscaleGenX0;
+#if defined(ENABLE_KERNELS) && !defined(_FULL_OPEN_SOURCE)
             CODECHAL_KERNEL_HEADER hmeDownscaleGenX1;
+#endif
             CODECHAL_KERNEL_HEADER hmeDownscaleGenX2;
+#if defined(ENABLE_KERNELS) && !defined(_FULL_OPEN_SOURCE)
```

Comment:
yes, it could be gather together, but due to other projects impact, it will not be included in this change.  Would rather to refine from windows side.